package.json

```diff
@@ -9,7 +9,18 @@
     "test": "lerna run test --stream --concurrency 1",
     "lint": "eslint **/*.js -c .eslintrc.json && markdownlint **/*.md --ignore **/node_modules/**",
     "screenshots": "lerna run screenshots --stream --concurrency 1",
-    "start": "lerna run bundle && npm start --prefix ./packages/d3fc"
+    "start": "lerna run bundle && npm start --prefix ./packages/d3fc",
+    "commitlint": "commitlint --from HEAD~1 to HEAD --verbose"
+  },
+  "husky": {
+    "hooks": {
+      "prepare-commit-msg": [
```

Comment:
I'm not too familiar with git hooks - I guess this is intended to display a notice whenever someone commits? In this case, I think it should (1) indicate that they need to use conventional commits and (2) suggest that they use commitizen to generate them.

The way it is worded at the moment implies that they **must** use `git cz` to generate commits.

Reply:
Yes, this executes just before the vim (or your usual terminal editor) opens for you to edit the default commit message. In *nix world there's a nice command that will automatically redirect git commit calls to npx git-cz but unfortunately this doesn't work cross platform so I opted to just give a notice :) 

will update the message to be more clear! :)